pkg/kapp/app/interfaces.go

```diff
@@ -6,11 +6,13 @@ package app
 import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"time"
 )
 
 type App interface {
 	Name() string
 	Namespace() string
+	CreationTimestamp() *time.Time
```

Comment:
should this really be a pointer?

Reply:
Was initially made it pointer to return nil from `func (a *LabeledApp) CreationTimestamp() *time.Time { return nil }` but we can return the time zero value here started doing that.